src/platform/packages/shared/kbn-lens-common/visualizations/layer_type_tab_display_name.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { i18n } from '@kbn/i18n';
+
+import type { LensLayerType } from './types';
+
+// Mapping of lens layer types to tab display names
+export const lensLayerTypeTabDisplayNames = {
```

Comment:
nit: I think we can remove this import because it is only used here

```suggestion
const lensLayerTypeTabDisplayNames = {
```